components/calendar/src/calendar_arithmetic.rs

```diff
@@ -7,7 +7,7 @@ use core::convert::TryInto;
 use core::marker::PhantomData;
 use tinystr::tinystr;
 
-#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq, Ord, PartialOrd)]
```

Comment:
I would ask you not to make this require `C: PartialOrd + Ord`, but I guess that ship has sailed with `Hash, Eq, PartialEq`.

Reply:
Agree on both counts.